### PR TITLE
Introduce ARM Fabric AzCLI

### PR DESCRIPTION
**Pull request by badeamarjieh**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by kairu-ms** [comment]
Please fix ci issues.

File: src/fabric/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Fabric Extension #
+This is an extension to Azure CLI to manage Fabric resources.
+
```
Comment (kairu-ms) [inline src/fabric/README.md:3]: Please fill this file

File: src/microsoft-fabric/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Azure CLI Fabric Extension #
+This is an extension to Azure CLI to manage Fabric resources.
+
```
Comment (kairu-ms) [inline src/microsoft-fabric/README.md:3]: Please fill this doc.

**Comment by kairu-ms** [comment]
Please register your extension in this doc https://github.com/Azure/azure-cli-extensions/blob/main/src/service_name.json

File: src/microsoft-fabric/azext_microsoft_fabric/aaz/latest/fabric/capacity/_create.py [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "fabric capacity create",
+)
+class Create(AAZCommand):
+    """Create a FabricCapacity
```
Comment (kairu-ms) [inline src/microsoft-fabric/azext_microsoft_fabric/aaz/latest/fabric/capacity/_create.py:18]: Please add examples for those commands

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**